tsl/src/compression/simple8b_rle_bitmap.h

```diff
@@ -0,0 +1,189 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+
+/*
+ * This is a specialization of Simple8bRLE decoder for bitmaps, i.e. where the
+ * elements are only 0 and 1. It also counts the number of ones.
+ */
+#include "compression/simple8b_rle.h"
+#pragma once
```

Comment:
Nit: As far as I know, `#pragma once` could be refined in an include. So, it is considered a good style to use it before including other files.

In this PR, we have header files that only use `#pragma once` and header files with `#pragma once` + include guard. I would suggest unifying that.

Reply:
Fixed the order.